src/org/intellij/grammar/editor/BnfSyntaxHighlighter.java

```diff
@@ -27,7 +27,8 @@ class BnfSyntaxHighlighter extends SyntaxHighlighterBase {
   public static final TextAttributesKey PATTERN = createTextAttributesKey("BNF_PATTERN", DefaultLanguageHighlighterColors.INSTANCE_FIELD);
   public static final TextAttributesKey NUMBER = createTextAttributesKey("BNF_NUMBER", DefaultLanguageHighlighterColors.NUMBER);
   public static final TextAttributesKey KEYWORD = createTextAttributesKey("BNF_KEYWORD", DefaultLanguageHighlighterColors.MARKUP_ATTRIBUTE);
-  public static final TextAttributesKey TOKEN = createTextAttributesKey("BNF_TOKEN", DefaultLanguageHighlighterColors.STRING);
+  public static final TextAttributesKey EXPLICIT_TOKEN = createTextAttributesKey("BNF_EXPLICIT_TOKEN", DefaultLanguageHighlighterColors.STRING);
```

Comment:
changing the identifier will break any customizations users already have, please keep `"BNF_TOKEN"` as ID

Reply:
fixed